cmd/term-check/main.go

```diff
@@ -14,8 +14,15 @@ var filepath = flag.String("config", "config.yaml", "Location of the configurati
 
 func main() {
 	zerolog.TimeFieldFormat = ""
+	debug := flag.Bool("debug", false, "sets log level to debug")
```

Comment:
Quick Env config could be like this:
```
debug := flag.Bool("debug", os.Getenv("LOG_LEVEL") == "debug", "sets log level to debug")
```

Reply:
I thought `flag` was specifically for command line flags? Do you still need to use `-debug` as a program argument?

Reply:
yes for command line flags, but you can have defaults set from ENV.
no need to explicitly use `-debug` if set from ENV